index.js

```diff
@@ -97,6 +99,14 @@ shareButton.on("click", function () {
   shareButton.text("Copied to Clipboard!");
 });
 
+downloadButton.on("click", function () {
+  let link = document.createElement('a');
+  link.download = 'stonks.png';
```

Comment:
Can we make this download file name dynamic based on the input word they have? 
Something like `link.download = input.val()`?